dist/index.bundle.js

```diff
@@ -0,0 +1,780 @@
+/******/ 				Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
+/******/ 			}
+/******/ 			Object.defineProperty(exports, '__esModule', { value: true });
+/******/ 		};
+/******/ 	})();
+/******/ 	
+/************************************************************************/
+var __webpack_exports__ = {};
+// This entry need to be wrapped in an IIFE because it need to be isolated against other modules in the chunk.
+(() => {
+/*!**********************!*\
+  !*** ./src/index.js ***!
+  \**********************/
+__webpack_require__.r(__webpack_exports__);
+/* harmony import */ var _gen_html__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./gen_html */ "./src/gen_html.js");
+/* harmony import */ var _add_new__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ./add_new */ "./src/add_new.js");
+/* harmony import */ var _style_css__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/*! ./style.css */ "./src/style.css");
+
+
+
+// Add and delete button
+let listArr = [];
+
+const list = document.getElementById('list');
+
+const reload = () => {
+  if (localStorage.getItem('todayList')) {
+    const oldStorage = localStorage.getItem('todayList');
+    const newStorage = JSON.parse(oldStorage);
+    list.innerHTML = '';
+    listArr = newStorage;
+    (0,_gen_html__WEBPACK_IMPORTED_MODULE_0__.default)(list, listArr);
+  } else {
+    const defaultItem = [
+      {
+        description: 'Add your first task',
+        completed: false,
+        index: 1,
+      },
+    ];
+    const newStorage = JSON.stringify(defaultItem);
+    listArr = defaultItem;
+    localStorage.setItem('todayList', newStorage);
+    list.innerHTML = '';
+    (0,_gen_html__WEBPACK_IMPORTED_MODULE_0__.default)(list, JSON.parse(newStorage));
+  }
+  const checkboxes = document.querySelectorAll('.checkbox');
+  for (let i = 0; i < checkboxes.length; i += 1) {
+    checkboxes[i].addEventListener('change', () => {
+      reload();
+    });
+  }
+};
+
+window.onload = reload();
+
+const task = document.getElementById('add-item');
+const addNewBtn = document.getElementById('add-new-btn');
+addNewBtn.addEventListener('click', () => {
+  if (task.value.length > 0) {
+    listArr.push((0,_add_new__WEBPACK_IMPORTED_MODULE_1__.default)(task.value, listArr.length + 1));
+    const newStorage = JSON.stringify(listArr);
+    localStorage.setItem('todayList', newStorage);
+    task.value = '';
+    reload();
+  } else {
+    alert('Tasks must contain text.\nTry typing something into the "Add to your list..." input.');
+  }
+});
+
+const clrBtn = document.getElementById('remove-all');
+clrBtn.addEventListener('click', () => {
+  for (let i = 0; i < listArr.length; i += 1) {
+    if (listArr[i].completed) {
+      listArr.splice(i, 1);
+      listArr.forEach((task) => {
+        if (task.index > i) {
+          task.index -= 1;
+        }
+      });
+      i -= 1;
+    }
+  }
+  localStorage.setItem('todayList', JSON.stringify(listArr));
+  reload();
+});
+})();
```

Comment:
For this part, I would recommend you to type shorter lines of code for your projects. Following the DRY principle, you should make your code shorter and not too repetitive.

Reply:
Thanks for the suggestions, I will fix it.

Reply:
I appreciate the time.